quakemigrate/io/triggered_events.py

```diff
@@ -13,6 +13,7 @@
 import logging
 
 from obspy import UTCDateTime
+from datetime import time
```

Comment:
This is a standard library package so should be imported on line 14 (and separated from the obspy etc imports by a blank line).